poset/poset.go

```diff
@@ -188,7 +188,7 @@ func (p *Poset) ProcessEvent(e *inter.Event) (err error) {
 		if err != nil {
```

Comment:
- remove defer func and move the commit to the end.
- on err, add a debug log instead of quietly return.